src/app/shared/issue-tables/dropdownfilter.ts

```diff
@@ -38,6 +38,14 @@ export function applyDropdownFilter(filter: Filter, data: Issue[]): Issue[] {
       ret = ret && issue.issueOrPr === 'PullRequest';
     }
 
+    if (filter.milestones.some((milestone) => issue.milestone.title === 'Issue without a milestone')) {
+      ret = ret && issue.issueOrPr === 'Issue';
+    }
+
+    if (filter.milestones.some((milestone) => issue.milestone.title === 'PR without a milestone')) {
+      ret = ret && issue.issueOrPr === 'PullRequest';
+    }
+
```

Comment:
These lines are not really necessary. The following lines with filtering using `filter.milestones` is already sufficient to also filter in issues and PRs without milestones.